layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra/0001-test.patch

```diff
@@ -0,0 +1,32 @@
+From c640de5fcf7bbc1c99d70492f7c2e3d47cdbd995 Mon Sep 17 00:00:00 2001
```

Comment:
https://devtalk.nvidia.com/default/topic/1072005/jetson-tx2/intel-8265ngw-causing-quot-wifi-is-disabled-by-hardware-switch-quot-/post/5431622/#5431622 do you not need to give credit to original authors?

Reply:
Ah guess it's in the commit message at least.

Reply:
This is just a test PR to see if it works. I did not polish it in any way

Reply:
Sure, really though as long as the commit message has a pointer to the thread where the nvidia engineer pointed out what was needed.